sdk/keyvault/azure-keyvault-administration/tests/recordings/test_backup_client_async.test_full_backup_and_restore.yaml

```diff
@@ -1,308 +1,275 @@
 interactions:
 - request:
-    body: null
-    headers:
-      Accept:
-      - application/json
-      Content-Length:
-      - '0'
-      Content-Type:
-      - application/json
-      User-Agent:
-      - azsdk-python-keyvault-administration/4.0.0b2 Python/3.5.3 (Windows-10-10.0.19041-SP0)
-    method: POST
-    uri: https://managedhsm/backup?api-version=7.2-preview
-  response:
-    body:
-      string: ''
-    headers:
-      cache-control: no-cache
-      content-length: '0'
-      content-security-policy: default-src 'self'
-      content-type: application/json; charset=utf-8
-      strict-transport-security: max-age=31536000; includeSubDomains
-      www-authenticate: Bearer authorization="https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47",
-        resource="https://managedhsm.azure.net"
-      x-content-type-options: nosniff
-      x-frame-options: SAMEORIGIN
-      x-ms-server-latency: '0'
-    status:
-      code: 401
-      message: Unauthorized
-    url: https://chlowehsm.managedhsm.azure.net/backup?api-version=7.2-preview
-- request:
-    body: '{"storageResourceUri": "https://storname.blob.core.windows.net/containerbg7icm2wfp27ig3",
-      "token": "redacted"}'
+    body: '{"token": "redacted", "storageResourceUri": "https://storname.blob.core.windows.net/containerzlj4vufdfg55shp"}'
```

Comment:
Strange the client didn't elicit a challenge. Does this test clear the client's cache?

Reply:
Ah, it might not have. These tests don't have the same structure as other packages' and I forgot to check for that. I'll take a look, thanks!